src/main/java/frc/team2412/robot/commands/autonomous/AutonomousCommand.java

```diff
@@ -39,16 +46,59 @@
 
     public AutonomousCommand(DrivebaseSubsystem d) {
         drivebaseSubsystem = d;
-    }
 
-    public Command getAutonomousCommand(Trajectory exampleTrajectory) {
+        // Create normalSpeedConfig for trajectory
+        TrajectoryConfig normalSpeedConfig = new TrajectoryConfig(
+                AutonomousCommand.AutoConstants.MAX_SPEED_METERS_PER_SECOND,
+                AutonomousCommand.AutoConstants.MAX_ACCELERATION_METERS_PER_SECOND_SQUARED)
+                // Add kinematics to ensure max speed is actually obeyed
+                .setKinematics(AutonomousCommand.AutoConstants.driveKinematics);
+        TrajectoryConfig fastSpeedConfig = new TrajectoryConfig(
+                AutonomousCommand.AutoConstants.MAX_SPEED_METERS_PER_SECOND *2,
+                AutonomousCommand.AutoConstants.MAX_ACCELERATION_METERS_PER_SECOND_SQUARED*2)
+                // Add kinematics to ensure max speed is actually obeyed
+                .setKinematics(AutonomousCommand.AutoConstants.driveKinematics);
+        Trajectory trajectoryOne = TrajectoryGenerator.generateTrajectory(
+                new Pose2d(7.5, 1.9, Rotation2d.fromDegrees(0)),
+                List.of(),
+                new Pose2d(7.3, 1.1, Rotation2d.fromDegrees(-90)),
+                fastSpeedConfig);
+        Trajectory trajectoryTwo = TrajectoryGenerator.generateTrajectory(
+                new Pose2d(7.3, 1.1, Rotation2d.fromDegrees(180)),
+                List.of(),
+                new Pose2d(5.1, 1.8, Rotation2d.fromDegrees(180)),normalSpeedConfig
+        );
+        Trajectory trajectoryThree = TrajectoryGenerator.generateTrajectory(
+                new Pose2d(5.1, 1.8, Rotation2d.fromDegrees(180)),
+                List.of(),
+                new Pose2d(2.1, 1.3, Rotation2d.fromDegrees(180)),normalSpeedConfig
+        );
+        Trajectory trajectoryFour = TrajectoryGenerator.generateTrajectory(
+                new Pose2d(2.1, 1.3, Rotation2d.fromDegrees(0)),
+                List.of(),
+                new Pose2d(5, 2.7, Rotation2d.fromDegrees(0)),normalSpeedConfig
+        ); ;
 
         ProfiledPIDController thetaController = new ProfiledPIDController(
                 0.000000005, 0, 0, AutoConstants.K_THETA_CONTROLLER_CONSTRAINTS);
         thetaController.enableContinuousInput(-Math.PI, Math.PI);
-        exampleTrajectory.relativeTo(drivebaseSubsystem.getPoseAsPoseMeters());
-        SwerveControllerCommand swerveControllerCommand = new SwerveControllerCommand(
-                exampleTrajectory,
+        trajectoryOne.relativeTo(drivebaseSubsystem.getPoseAsPoseMeters());
+        trajectoryTwo.relativeTo(drivebaseSubsystem.getPoseAsPoseMeters());
```

Comment:
Is there a reason why this isn't repeated for the other two trajectories?

Reply:
Yes, the reason is I have a bad memory